src/app/shared/breadcrumbs/breadcrumbs.module.ts

```diff
@@ -0,0 +1,15 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { BreadcrumbsComponent } from './breadcrumbs.component';
+
+
+
+@NgModule({
+  declarations: [
+    BreadcrumbsComponent
+  ],
```

Comment:
add BreadcrumbsComponent to exports: [] as well to allow this component to use outside of current module

Reply:
> add BreadcrumbsComponent to exports: [] as well to allow this component to use outside of current module

added